src/app/components/SearchForm/TotalWorks.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import PropTypes from 'prop-types';
+import * as searchActions from '../../actions/SearchActions';
+
+
+class TotalWorks extends React.Component {
+  constructor(props) {
+    super(props);
+    const { dispatch } = props;
+    this.boundActions = bindActionCreators(searchActions, dispatch);
+  }
+
+  componentDidMount() {
```

Comment:
Should this be `componentWillMount`? I was getting errors that `this.props.totalWorks` was `undefined` in the `render` method

Reply:
Apparently a cache/storage issue, marking as resolved